pkg/jwt/decode_test.go

```diff
@@ -0,0 +1,61 @@
+package jwt
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+)
+
+type DecodeTestInput struct {
+	InputVal     string
+	ExpectVal    string
+	ExpectErr    bool
+	ExpectErrVal string
+}
+
+func TestDecode(t *testing.T) {
+
+	table := []DecodeTestInput{
+		{
```

Comment:
suggestions:
Do we want to add Description into test cases? Or is it gonna accomplish in another task?